keel-bakery-plugin/src/main/kotlin/com/netflix/spinnaker/keel/bakery/api/ImageSpec.kt

```diff
@@ -1,9 +1,12 @@
 package com.netflix.spinnaker.keel.bakery.api
 
+import com.netflix.spinnaker.keel.api.Applicationed
```

Comment:
spent a while trying to get the right english teacher words to call out `Application` -> `Applicationed` but I fell short so I suspect my dad is going to disown me (gerund? past particple? past gerund? maybe Applicationed is the past gerunded form of Application)

Maybe HasApplication as the interface instead? I dunno, naming is hard

Reply:
I like `HasApplication` FWIW.

Reply:
😆 changing to `HasApplication` because it seems less bad